build/scripts/lib.sh

```diff
@@ -77,6 +77,11 @@ if [[ $? -eq 0 ]]; then
     DOCKER="podman"
     PODMAN_OPTS="--security-opt label=disable --userns=keep-id"
 else
-    DOCKER="docker"
-    PODMAN_OPTS=""
+    if [[ $(docker --version | grep -q podman) -eq 0 ]]; then
```

Comment:
Sure, something like the below?

```
diff --git a/build/scripts/lib.sh b/build/scripts/lib.sh
index 82c9205de6f875..9c21e4d51e3463 100755
--- a/build/scripts/lib.sh
+++ b/build/scripts/lib.sh
@@ -72,7 +72,7 @@ function get_output_dir()
     return 0
 }
 
-grep "^NAME=Fedora$" /etc/os-release > /dev/null
+command -v podman > /dev/null
 if [[ $? -eq 0 ]]; then
     DOCKER="podman"
     PODMAN_OPTS="--security-opt label=disable --userns=keep-id"
```

This will be simpler, with the caveat that `podman` will be preferred over `docker` if it is available.